ios/BranchUniversalObject+RNBranch.m

```diff
@@ -0,0 +1,69 @@
+//
+//  BranchUniversalObject+RNBranch.m
+//  RNBranch
+//
+//  Created by Jimmy Dee on 1/26/17.
+//  Copyright © 2017 Dispatcher. All rights reserved.
+//
+
+#import "RCTLog.h"
+
+#import "BranchUniversalObject+RNBranch.h"
+#import "NSObject+RNBranch.h"
+#import "RNBranchProperty.h"
+
+@implementation BranchUniversalObject(RNBranch)
+
++ (NSDictionary<NSString *,RNBranchProperty *> *)supportedProperties
+{
+    static NSDictionary<NSString *, RNBranchProperty *> *_universalObjectProperties;
+    static dispatch_once_t once = 0;
+    dispatch_once(&once, ^{
+        _universalObjectProperties =
+        @{
+          @"canonicalUrl": [RNBranchProperty propertyWithSetterSelector:@selector(setCanonicalUrl:) type:NSString.class],
+          @"contentDescription": [RNBranchProperty propertyWithSetterSelector:@selector(setContentDescription:) type:NSString.class],
+          @"contentImageUrl": [RNBranchProperty propertyWithSetterSelector:@selector(setImageUrl:) type:NSString.class],
+          @"contentIndexingMode": [RNBranchProperty propertyWithSetterSelector:@selector(setContentIndexingMode:) type:NSString.class],
+          @"metadata": [RNBranchProperty propertyWithSetterSelector:@selector(setMetadata:) type:NSDictionary.class],
+          @"title": [RNBranchProperty propertyWithSetterSelector:@selector(setTitle:) type:NSString.class]
```

Comment:
Currently, there are more properties for Branch Universal Object in the native SDK (https://github.com/BranchMetrics/ios-branch-deep-linking/blob/master/Branch-SDK/Branch-SDK/BranchUniversalObject.h) Are there plans to add them later?